src/toolbox/areaOfInterestTemplate.js

```diff
@@ -149,7 +149,24 @@ function aoiListTemplate() {
                     <i class="question circle outline icon"></i>
                 </button>
             </div>
-
+            <div class="ngm-aoi-input-container">
+                <label>${i18next.t('tbx_name_label')}:</label>
+                <div class="ui mini input">
+                  <input
+                    class=${`ngm-aoi-name-input-${index}`}
+                    type="text" .value="${i.name}"
```

Comment:
Since the value is a text, can we simply right? :
```suggestion
                    type="text" value="${i.name}"
```

Reply:
dot required for correct cancel work